### PR TITLE
ticdc: update open protocol

### PR DESCRIPTION
**Pull request by leoppro**
### What is changed, added or deleted? (Required)
update open protocol

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by leoppro** [comment]
@amyangfei @liuzix  PTAL

File: ticdc/ticdc-open-protocol.md [hunk 1]
```diff
@@ -276,36 +276,33 @@ COMMIT;
...
+| FLOAT                 | 4    | {"t":4,"v":153.123} | |
+| DOUBLE                | 5    | {"t":5,"v":153.123} | |
+| NULL                  | 6    | {"t":6,"v":null} | |
+| TIMESTAMP             | 7    | {"t":7,"v":"1973-12-30 15:30:00"} | |
+| BIGINT                | 8    | {"t":8,"v":123} | |
+| MEDIUMINT             | 9    | {"t":9,"v":123} | |
+| DATE                  | 10/14   | {"t":10,"v":"2000-01-01"} | |
+| TIME                  | 11   | {"t":11,"v":"23:59:59"} | |
+| DATETIME              | 12   | {"t":12,"v":"2015-12-20 23:58:58"} | |
+| TEAR                  | 13   | {"t":13,"v":1970} | |
```
Comment (amyangfei) [inline ticdc/ticdc-open-protocol.md:293]:
```suggestion
| YEAR                  | 13   | {"t":13,"v":1970} | |
```

File: ticdc/ticdc-open-protocol.md [hunk 2]
```diff
@@ -276,36 +276,33 @@ COMMIT;
...
+| BIT                   | 16   | {"t":16,"v":81} | |
+| JSON                  | 245  | {"t":245,"v":"{\\"key1\\": \\"value1\\"}"} | |
+| DECIMAL               | 246  | {"t":246,"v":"129012.1230000"} | |
+| ENUM                  | 247  | {"t":247,"v":1} | |
+| SET                   | 248  | {"t":248,"v":3} | |
+| TINTTEXT/TINTBLOB     | 249  | {"t":249,"v":"5rWL6K+VdGV4dA=="} | value 编码为 Base64 |
+| MEDIUMTEXT/MEDIUMBLOB | 250  | {"t":250,"v":"5rWL6K+VdGV4dA=="} | value 编码为 Base64 |
+| LONGTEXT/LONGBLOB     | 251  | {"t":251,"v":"5rWL6K+VdGV4dA=="} | value 编码为 Base64 |
+| TEXT/BLOB             | 252  | {"t":252,"v":"5rWL6K+VdGV4dA=="} | value 编码为 Base64 |
+| CHAT/BINARY           | 254  | {"t":254,"v":"测试"} / {"t":254,"v":"\\x89PNG\\r\\n\\x1a\\n"} | value 编码为 UTF-8；当上游类型为 BINARY 时，将对不可见的 ASCII 字符转义 |
```
Comment (amyangfei) [inline ticdc/ticdc-open-protocol.md:304]: is s/CHAT/CHAR ?

**Review by overvenus — APPROVED** [review]
LGTM

File: ticdc/ticdc-open-protocol.md [hunk 3]
```diff
@@ -276,36 +276,33 @@ COMMIT;
...
```
Comment (leoppro) [inline ticdc/ticdc-open-protocol.md:304]:
```suggestion
| CHAR/BINARY           | 254  | {"t":254,"v":"测试"} / {"t":254,"v":"\\x89PNG\\r\\n\\x1a\\n"} | value 编码为 UTF-8；当上游类型为 BINARY 时，将对不可见的 ASCII 字符转义 |
```

**Comment by TomShawn** [comment]
@leoppro 如果提了有关 TiCDC GA 发版的文档 PR，请@我。否则 PR 无法及时得到语言 review 和合，可能导致 TiCDC GA 所需的文档无法按时到位。

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by leoppro** [comment]
> @leoppro 如果提了有关 TiCDC GA 发版的文档 PR，请@我。否则 PR 无法及时得到语言 review 和合并，**可能导致 TiCDC GA 所需的文档无法按时到位，影响交付**。 所以如果明天 GA 前还有中文 PR 要提上来，**请务必@我**。

好的

**Pull request merged**